index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Introduction to SimPEG using an example with electrical resistivity tomography
+---
+
++++ { "part": "abstract" }
+
```

Comment:
Geophysical inversions are used to build models of the subsurface from data collected over the Earth. Solving an inverse problem and obtaining a meaningful result requires bringing together numerical tools for solving partial differential equations, formulating an optimization problem, and specifying tuning parameters and heuristics for performing the optimization. All of these choices are what make an inversion work (or not) in practice. These are captured in the software that implements the inversion.

Reply:
Geophysical inversions are used to build models of the subsurface from data collected over the Earth. Solving an inverse problem and obtaining a meaningful result requires bringing together numerical tools for solving partial differential equations, formulating an optimization problem, and specifying tuning parameters and heuristics for performing the optimization. All of these choices are what make an inversion work (or not) in practice. These are captured in the software that implements the inversion.